database/connect_to_database.py

```diff
@@ -9,25 +9,27 @@
 
 
 def get_attribute(key: str):
-    filename = "../config.json"
+    filename = "./config.json"
     if not isfile(filename):
+        print("Problem with file")
```

Comment:
Sysopy vibes ❤️

Reply:
Dobra to się usunie bo zostało jak szukałem czemu nie działa